components/edit-collective/Form.js

```diff
@@ -109,6 +109,10 @@ class EditCollectiveForm extends React.Component {
         defaultMessage:
           'The legal name is private and shared with the hosts for donation receipts, tax forms and when you submit and expense. This name is not displayed publicly and it must be your legal name.',
       },
+      examples: {
+        id: 'examples',
+        defaultMessage: 'e.g. {examples}',
```

Comment:
```suggestion
        defaultMessage: 'e.g., {examples}',
```

Reply:
We have 14 places using `e.g.` right now, and none using `e.g,` - including places where that Alanna updated. I'm gonna strive for consistency for now and ask for feedback on Slack. We'll update everything if needed.